### PR TITLE
KCE-72 Remove DDS from kpsr-robotics

### PR DESCRIPTION
**Pull request by joseluisbf-kpsr**
[KCE-72]

Tested with DEB `kpe-core-dev` v7.11.1 (not kpsr-core v7.8.1, no DEB packages for CE) and `kpe-core-zmq-dev` v8.0.0 (no v8 in CE): it will build for upcoming versions.

### TODO
- `api-doc/` → unchanged as in kpsr-core

[KCE-72]: https://klepsydra.atlassian.net/browse/KCE-72?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: geometry/tests/zmq_geometry_tests/src/zmqstg_geometry_test.cpp [hunk 1]
```diff
@@ -68,10 +68,10 @@ TEST(ZmqGeometryTest, ZmqGeometryTest)
         _fromZmqMiddlewareProvider
             .getJsonFromMiddlewareChannel<kpsr::geometry::PoseEventData>(subscriber, 1);
 
-    kpsr::Publisher<kpsr::geometry::PoseEventData> *poseEventDataToDDSChannel =
+    kpsr::Publisher<kpsr::geometry::PoseEventData> *poseEventDataToChannel =
```
Comment (joseluisbf-kpsr) [inline geometry/tests/zmq_geometry_tests/src/zmqstg_geometry_test.cpp:71]: Appropriate renaming?

Comment (mharshe) [reply geometry/tests/zmq_geometry_tests/src/zmqstg_geometry_test.cpp:71]: `poseEventDataToZmqChannel` instead

File: uav_sensors/tests/zmq_uav_sensors_tests/src/zmqstg_sensors_test.cpp [hunk 2]
```diff
@@ -68,11 +68,10 @@ TEST(ZmqSensorsTest, ZmqSensorsTest)
         _fromZmqMiddlewareProvider
             .getJsonFromMiddlewareChannel<kpsr::sensors::LaserScanEvent>(subscriber, 1);
 
-    kpsr::Publisher<kpsr::sensors::LaserScanEvent> *laserScanEventToDDSChannel =
+    kpsr::Publisher<kpsr::sensors::LaserScanEvent> *laserScanEventToChannel =
```
Comment (mharshe) [inline uav_sensors/tests/zmq_uav_sensors_tests/src/zmqstg_sensors_test.cpp:71]: laserScanEventToZMQChannel

Comment (joseluisbf-kpsr) [reply uav_sensors/tests/zmq_uav_sensors_tests/src/zmqstg_sensors_test.cpp]: laserScanEventToZ**mq**Channel

**Comment by mharshe** [comment]
> Tested with DEB kpe-core-dev v7.11.1 (not kpsr-core v7.8.1, no DEB packages for CE) and kpe-core-zmq-dev v8.0.0 (no v8 in CE): it will build for upcoming versions.

It should only be tested with latest kpsr-core (not kpe-core). This is already being done by the CI.

**Comment by joseluisbf-kpsr** [comment]
> > Tested with DEB kpe-core-dev v7.11.1 (not kpsr-core v7.8.1, no DEB packages for CE) and kpe-core-zmq-dev v8.0.0 (no v8 in CE): it will build for upcoming versions.
> 
> It should only be tested with latest kpsr-core (not kpe-core). This is already being done by the CI.

2x extras for free :D

**Comment by joseluisbf-kpsr** [comment]
Dependency to kpsr-core v8.0.0 will break CI until [KCE-73] ([PR](https://github.com/klepsydra-technologies/kpsr-core/pull/97)).

**Comment by mharshe** [comment]
Ok to squash & approve

**Comment by joseluisbf-kpsr** [comment]
> Ok to squash & approve

Tag as v3.2.1 or upper?

**Comment by mharshe** [comment]
tag as v4.0.0 to ensure that it is clear that the change is big (dds removed !)

**Review by mharshe — APPROVED** [review]

**Pull request merged**